pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -45,8 +47,45 @@ import (
 	registryconfig "github.com/openshift/origin/pkg/dockerregistry/server/configuration"
 )
 
+var prune = flag.Bool("prune", false, "prune blobs from the storage and exit")
+
+// ExecutePruner runs the pruner.
+func ExecutePruner(configFile io.Reader) {
+	log.Infof("prune version=%s", version.Version)
```

Comment:
The distribution version doesn't say much. Can this output OpenShift release/git version?